lib/CL/clGetPlatformIDs.c

```diff
@@ -37,115 +37,115 @@
 */
 #ifdef BUILD_ICD
 struct _cl_icd_dispatch pocl_dispatch = {
-  &POname(clGetPlatformIDs),
```

Comment:
Can you revert this large style change for now in this commit? Is it according to the C formatter?
Let's avoid large white-space/style-only commits but fix the style gradually with other changes. So, just submit your additional functionality, not the large style fix, thanks.

Reply:
The white space was added by the format-last-commit.sh script. I reverted the style changes.